modules/imgproc/src/color_lab.cpp

```diff
@@ -1571,6 +1573,69 @@ struct RGB2Lab_b
         }
     }
 
+#if CV_NEON
```

Comment:
It should be `#if CV_SIMD`.

Reply:
Hi @asmorkalov  - this code is specific to 128-bit. If we replace `CV_NEON` with `CV_SIMD`, the version supporting wider vectors at line 1692 will stop running.

Would replacing `CV_NEON` with `CV_SIMD128` a better option?

Reply:
@asmorkalov `CV_SIMD` is below. This is dedicated branch for NEON only (existed `v_lut()`-based approach is not very fast with NEON).